services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
This line of code can raise a `MovieSession.DoesNotExist` exception if the session with the provided id doesn't exist. It's a good practice to handle this exception and provide a user-friendly message. You can do this by using Django's `get_object_or_404()` function or by adding a `try/except` block around this line.